src/monit.c

```diff
@@ -804,7 +807,7 @@ static void help() {
                " -I            Do not run in background (needed for run from init)\n"
                " --id          Print Monit's unique ID\n"
                " --resetid     Reset Monit's unique ID. Use with caution\n"
-               " -B            Batch command line mode (do not output tables or colors)\n"
+               " -B            Turn off batch command line mode (output tables and colors)\n"
```

Comment:
It would make more sense to phrase this as 'Turn on COLOUR-POWER mode' (or similar) than 'turn off batch command line mode' since batch command line mode doesn't really feel like a thing you turn off. Anyway, whatever, I don't really care.